src/dispatch-payload.ts

```diff
@@ -103,6 +103,11 @@ export const DispatchPayload = async (
       } else {
         console.warn(handlerError.message);
       }
+    } else if (isAllowNetError(handlerError)) {
+      console.warn(
+        "⚠️   deno-slack-runtime: Detected missing network permissions. Add the domain to your manifest's `outgoingDomains` to resolve the `--allow-net` error.",
+      );
+      throw handlerError;
```

Comment:
Is there a reason we want to throw the Deno error instead of throwing this warning message as an error? We have a couple other instances in this repo where we throw errors with custom messages, but if we think the stack will be useful in debugging I can see why we'd want to keep it. Maybe a Custom Error Message in that case may be a nice touch?

I worry that the devs eye will be drawn to the error instead of the warning and they'll waste time trying to resolve the `--allow-net` problem before seeing the warning and realizing that's the actual solution.

Reply:
now updates the message